.circleci/config.yml

```diff
@@ -22,21 +22,7 @@ jobs:
       - checkout
       - restore_cache:
           key: sbtcache
-      - run: sbt ++2.12.14! core/test http4s/test akkaHttp/test play/test zioHttp/test examples/compile catsInterop/compile benchmarks/compile tools/test codegenSbt/test clientJVM/test monixInterop/compile tapirInterop/test federation/test reporting/test
-      - save_cache:
-          key: sbtcache
-          paths:
-            - "~/.ivy2/cache"
-            - "~/.sbt"
-            - "~/.m2"
-  test212_jdk8:
```

Comment:
sttp 3.6 requires at least Java 11

Reply:
that's pity, what happens with Java 11? does it fail at runtime?

Reply:
I would have expected a `UnsupportedClassVersionError`, but there doesn't seem to be one, so I have restored the test.